appveyor.yml

```diff
@@ -28,6 +28,7 @@ build_script:
   - C:\Ruby21\DevKit\devkitvars.bat
   - echo %PATH%
   - bundle exec rake build
+  - bundle exec rake build64
```

Comment:
You can't do this. The compiler toolchain is 32-bit. You need to add another ruby_version variable to the matrix section. 
I think it will look like `- ruby_version: "200-x64"`